doc/nrf/ug_nrf5340.rst

```diff
@@ -58,16 +58,10 @@ When the MCU boots, it always starts executing from the secure area.
 
 In Zephyr, the firmware of the application core is built using one of the following build targets:
 
-* ``nrf5340dk_nrf5340_cpuapp`` for the secure domain.
-* ``nrf5340dk_nrf5340_cpuapp_ns`` for the non-secure domain.
-  On selecting this build target, the build system includes an additional secure firmware component before building the main firmware on the non-secure domain.
-  The additional component is :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
+* ``nrf5340dk_nrf5340_cpuapp`` for only the secure domain.
+* ``nrf5340dk_nrf5340_cpuapp_ns`` for adding the non-secure domain.
```

Comment:
Not sure if I feel that "adding" sits well with me. It's just a feeling, without me being completely able to say why, so I will leave this completely up to you to decide if you want to change or not. 
The alternative would maybe be to write "for both the secure and non-secure domain